### DISCUSSION TITLE
[v5] 页面上有多个chart时slider拖拽出现问题

### DISCUSSION THREAD

**Discussion by yshenhua** [antvis/G2#5198] [Q&A]
Demo：https://codesandbox.io/s/zen-borg-c4ztt2?file=/src/index.js

slider向右拖拽时鼠标遇到右边的另一个chart后会突然跳到左边，向左拖拽时则相反。

**Comment by pearmini** [comment]
现在还有问题吗？我看线上 demo 似乎正常了？
